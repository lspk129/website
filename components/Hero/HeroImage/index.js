import styled from 'styled-components';

const HomeImage = styled.div`
  // background-image: url("http://trendytheme.net/demo2/wp/69/multipage/wp-content/uploads/revslider/web-product-dark/webproduct_darkbg2.jpg");
  background: linear-gradient(210deg, #318EDF, #46c340);
  background-position: center center;
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  font-family: Roboto;
  height: 100vh;
  display: grid;
  grid-template-columns: 20% 60% 20%;
  grid-template-rows: 20% 25% 15% 15% 1fr;
`;

export default HomeImage;
