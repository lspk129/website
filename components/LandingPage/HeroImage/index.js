import styled from 'styled-components';

export default styled.div`
  background-image: url("http://trendytheme.net/demo2/wp/69/multipage/wp-content/uploads/revslider/web-product-dark/webproduct_darkbg2.jpg");
  background-position: center center;
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  margin: -10px;
  height: 100vh
`;

