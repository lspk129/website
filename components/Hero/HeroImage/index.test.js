import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import HeroImage from './index';

describe('HeroImage', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<HeroImage />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    const wrapper = shallow(<HeroImage />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('div');
  });
});
