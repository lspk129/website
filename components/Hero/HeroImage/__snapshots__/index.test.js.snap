// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroImage renders correctly 1`] = `
<div
  className="sc-bdVaJa bcrzSP"
/>
`;
